src/EventListener/ResizeFormListener.php

```diff
@@ -68,7 +68,10 @@ public function preSetData(FormEvent $event): void
         }
 
         // First remove all rows except for the prototype row
-        // Type cast to string, because Symfony form can returns integer keys
+        // Type cast to string, because Symfony form can return integer keys
+        /**
+         * @psalm-suppress RedundantCastGivenDocblockType
+         */
```

Comment:
This will be corrected in SF 6.2 with
https://github.com/symfony/symfony/pull/46813

Might be interesting to add a comment

Reply:
😕  mmmm I guess it's already using SF 6.2?

Reply:
I mean, the cast is only needed to fix issue on SF < 6.2. (I think test were added for this. One way to check would be to run test locally on 6.2 without the cast and see if the ci is green while it's red on symfony 5.4.)

When we'll drop version < 6.2 we will be able to remove the cast and the psalm-suppress.